tests/lib/nrf_modem_lib/at_cmd_filter/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(nrf_modem_lib)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+
+target_include_directories(app
+  PRIVATE
+  ${ZEPHYR_BASE}/../nrf/include/modem/
+)
```

Comment:
Why the new lines lines here ?
Your line doesn't seem to be that long.

Also use: `${ZEPHYR_NRF_MODULE_DIR}/include/modem/` instead.